bin/posydon-run-pipeline

```diff
@@ -407,6 +413,12 @@ def train_interpolators(i, path_to_csv_file, verbose=False):
                                    (type(grid.final_values[key][0]) != np.str_)
                                    and any(~np.isnan(grid.final_values[key]))
                                    and f"S{i}_{MODEL_NAME}" in key)]
+            
+            out_nan_keys = [key for key in grid.final_values.dtype.names
+                            if type(grid.final_values[key][0]) != np.str_
+                            and all(np.isnan(grid.final_values[key]))
+                        ]
```

Comment:
Here we need to request that the values belong to the supernova models. We don't want to include here stuff which are only keys for the mass transfer classes.

Reply:
fixed